content/company/strategy/code-graph/code-intelligence/index.md

```diff
@@ -45,15 +45,16 @@ At the moment maturity varies depending on the area of ownership:
 
 - **Language indexers:** Maturity varies widely per language. See [LSIF indexers documentation](https://docs.sourcegraph.com/code_intelligence/references/indexers).
 
-- **Code intelligence platform:** Viable with a solid foundation, we're currently working on scaling and improving performance to push the limits of precise indexing and build the Global Code Graph.
+- **Code intelligence platform:** We're currently focusing on delivering an out-of-the-box precise code intelligence solution to a small number of eager customers.
 
-- **Code navigation:** New ownership area. Basic features are implemented but have a considerable amount of debt and room for improvement.
+- **Code navigation:** Baseline features are implemented but have a considerable amount of debt and room for improvement. We're currently focusing on researching and implementating features that will make the overall navigation experience snappier, more intuitive and complete.
 
 In the last few months we’ve:
 
 - Added precise support for the JVM ecosystem (Java, Scala and soon Kotlin).
-- Built the foundation for our future scaling, as we’re planning to index an exponential amont of repositories and their dependencies.
-- Ran successful POCs which paved the way to implementing out-of-the-box cross-repository and cross-dependency navigation.
+- Added search-based support for Apex.
+- Shipped critical platform features necessary to set up and run auto-indexing at on-prem customers that use GCP or AWS.
+- Taken our first steps into improving the code navigation experience by tackling longstanding papercuts.
```

Comment:
Do we want a _such as_ example here?

Reply:
If there's a papercuts label you could link to closed issues with it, maybe.

Reply:
We have the [Papercuts discussion board](https://github.com/sourcegraph/sourcegraph/discussions/24905?sort=top ) but I'm unsure if that would be helpful or not.